examples/amqp-quickstart/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -13,8 +13,10 @@ smallrye.messaging.sink.data.durable=true
 smallrye.messaging.sink.my-channel.connector=smallrye-amqp
 smallrye.messaging.sink.my-channel.address=hello
 smallrye.messaging.sink.my-channel.durable=true
+smallrye.messaging.sink.my-channel.client-options-name=my-topic-config
 
 # AMQP source (we read from)
 smallrye.messaging.source.my-topic.connector=smallrye-amqp
 smallrye.messaging.source.my-topic.address=hello
 smallrye.messaging.source.my-topic.durable=true
+smallrye.messaging.source.my-topic.client-options-name=my-topic-config
```

Comment:
It is on purpose you don't use the second one?

Reply:
oh, missed this, I was playing with it and just leaved it like this, I'll change it